plaidcloud/rpc/remote/rpc_common.py

```diff
@@ -164,10 +166,10 @@ def clean_args(arg_dict):
                     # We don't want to clean queries or UDF code, as they can legally include
                     # > and <. Bleach was a little overzealous, as it was replacing things like &.
                     if isinstance(arg_dict[arg], string_types) and arg not in ["query", "code"]:
```

Comment:
Should we continue to have exceptions for query and code fields now that the search is very specific?  If this might pick up something other than <script> then let's leave the exceptions but hoping we don't need to leave any gates open.

Reply:
I went back and forth on this a bit. I don't see script tags doing any harm in the database, and there may be times people want to do some HTML templating in UDFs. They're very niche use cases that we might never hit, so if we want to just get rid of script tags everywhere, that's fine.